src/include/uapi/ipc/topology.h

```diff
@@ -193,11 +196,13 @@ struct sof_ipc_comp_tone {
 	int32_t ramp_step;
 } __attribute__((packed));
 
-/** \brief Types of processing components */
-enum sof_ipc_process_type {
-	SOF_PROCESS_NONE = 0,		/**< None */
-	SOF_PROCESS_EQFIR,		/**< Intel FIR */
-	SOF_PROCESS_EQIIR,		/**< Intel IIR */
+/** \brief Types of EFFECT */
+enum sof_ipc_effect_type {
+	SOF_EFFECT_NONE = 0,		/**< None */
+	SOF_EFFECT_INTEL_EQFIR,		/**< Intel FIR */
+	SOF_EFFECT_INTEL_EQIIR,		/**< Intel IIR */
+	SOF_PROCESS_KEYWORD_DETECT,     /**< Keyword Detection */
+	SOF_PROCESS_VOICE_DETECT,       /**< Voice Detection */
```

Comment:
And the ipc_process_type is removed, so it reverts what the previous patch did? this looks quite wrong...